Dev#100328018

jeong760 opened the pull request:
@bonesoul,

Could you review and merged this PR? 
I've added some configurations file and improved a code.

Config
- config-example.json

Config/Coins
- globaltoekn.json

web/default/Content/img/coins/icon
- GLT.png

Config/Pools
- advanced-example.json
- bitcoin-example.json
- config-example.json
- default-example.json

Improve or correct
- Pool hash rate corrected.
- Added a code for Getinfo() and Getnetworkinfo() running based on wallet version.

Regards,
John Ahn

jeong760 committed:
Add files via upload

jeong760 committed:
Add files via upload

jeong760 committed:
Update default.json

jeong760 committed:
Update pool.json

jeong760 committed:
Add files via upload

jeong760 committed:
Correct a pool hash rate issue.

jeong760 committed:
Add code support a Geinfo and Getnetworkinfo by wallet version

jeong760 committed:
Add files via upload

jeong760 committed:
Fixed a Use patten matching Message and add a string CoinVersion

jeong760 committed:
Create GetAddressInfo.cs

jeong760 committed:
Removed a white space on paymentprocessor.cs

jeong760 committed:
Fixed a version:null issue on console and add getaddressinfo()

jeong760 committed:
WIP

jeong760 committed:
Test a IsMine result by the command of coins wallet supported.

jeong760 committed:
Added a test code for _daemonClient.GetAddressInfo

jeong760 committed:
Change back code for GetAddressInfo()

jeong760 committed:
Update SerializersTests.cs

- commented a // _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true }); related with GetAddressInfo()

jeong760 committed:
Update JobTests.cs

- commented a // _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true }); related with GetAddressInfo()

jeong760 committed:
Update ShareTests.cs

- commented a // _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true }); related with GetAddressInfo()

jeong760 committed:
Update GenerationTransactionTests.cs

- commented a // _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true }); related with GetAddressInfo()

jeong760 committed:
Update OutputsTests.cs

- commented a // _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true }); related with GetAddressInfo()

jeong760 committed:
Update HybridStorage.Shares.cs

Change code to following for pool hashrate.

From 
_redisProvider.Client.HIncrByFloat(currentKey, miner.Username, (double)miner.Difficulty);

To
_redisProvider.Client.HIncrByFloat(currentKey, miner.Username, (double)share.Difficulty);

Regards,
John Ahn

globaltoken committed:
Updated GetAddressInfo for valid output

Updated GetAddressInfo for valid output

globaltoken committed:
Updated GetAddressInfo

Updated GetAddressInfo

globaltoken committed:
GetAddressInfo activated

GetAddressInfo activated

globaltoken committed:
Updated to new format

Updated PaymentProcessor.cs to the new Wallet getaddressinfo format. It still should be compatible for old wallets.

jeong760 committed:
Merge pull request #1 from globaltoken/develop

Updated to the new format.

globaltoken committed:
added rpcupdate arg

rpcupdate argument added, to auto use getaddressinfo instead of the old validateaddress rpc

globaltoken committed:
Added RpcUpdate arg

Added RpcUpdate arg

globaltoken committed:
Bug fix for older coins

globaltoken committed:
Merge pull request #1 from jeong760/develop

Merge pull request #1 from globaltoken/develop

jeong760 committed:
Merge pull request #2 from globaltoken/develop

Bugfix I

globaltoken committed:
Update ICoinConfig.cs

globaltoken committed:
Update CoinConfig.cs

globaltoken committed:
Update CoinConfig.cs

jeong760 committed:
Merge pull request #3 from globaltoken/develop

Bugfix II

globaltoken committed:
changed to bool?

jeong760 committed:
Merge pull request #4 from globaltoken/develop

changed to bool?

globaltoken committed:
bool? change

jeong760 committed:
Merge pull request #5 from globaltoken/develop

bool? change II

globaltoken committed:
Update PaymentProcessor.cs

globaltoken committed:
Update globaltoken.json

jeong760 committed:
Merge pull request #6 from globaltoken/develop

Bugfixes

globaltoken committed:
var should be set as bool, not string

globaltoken committed:
Update CoinConfig.cs

globaltoken committed:
Update CoinConfig.cs

globaltoken committed:
Update ICoinConfig.cs

globaltoken committed:
Update PaymentProcessor.cs

jeong760 committed:
Merge pull request #7 from globaltoken/develop

Final working Version

jeong760 commented:
Everything is wroks fine. please update a this PR

jeong760 committed:
Update SerializersTests.cs

Enabled a GetAddressInfo() test code.
-  _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true });

jeong760 committed:
Update JobTests.cs

Enabled a GetAddressInfo() test code.
-  _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true });

jeong760 committed:
Update ShareTests.cs

jeong760 committed:
Update GenerationTransactionTests.cs

Enabled a GetAddressInfo() test code.
-  _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true });

jeong760 committed:
Update OutputsTests.cs

Enabled a GetAddressInfo() test code.
-  _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true });

jeong760 committed:
Update SerializersTests.cs

_daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { RpcUpdate = true });

jeong760 committed:
Update SerializersTests.cs

_daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsMine = true });

jeong760 committed:
Update SerializersTests.cs

//_daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true });

jeong760 committed:
Update JobTests.cs

//_daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true })

jeong760 committed:
Update ShareTests.cs

// _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true });

jeong760 committed:
Update GenerationTransactionTests.cs

// _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true });

jeong760 committed:
Update OutputsTests.cs

// _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true });

jeong760 committed:
Update SerializersTests.cs

// _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true });

jeong760 committed:
Update JobTests.cs

// _daemonClient.GetAddressInfo(Arg.Any<string>()).Returns(new GetAddressInfo { IsValid = true });

jeong760 committed:
Delete and update a new  GLT logo

jeong760 committed:
Replaced  a GLT  logo as new.

Added a new GLT logo

jeong760 committed:
Delete GLT.png

jeong760 committed:
Add files via upload

jeong760 committed:
Delete GLT.png

jeong760 committed:
Add files via upload

GLT logo size change to 32 pixels  by 32 pixels

jeong760 committed:
Delete GLT.png

jeong760 committed:
Add files via upload

jeong760 committed:
Delete GLT.png

jeong760 committed:
Add files via upload

jeong760 committed:
Delete GLT.png

jeong760 committed:
Add files via upload

Changed GLT logo size to 32 by 32 pixels.

jeong760 committed:
Update GenerationTransaction.cs

bonesoul commented on src/CoiniumServ/Transactions/GenerationTransaction.cs:
```diff
@@ -158,7 +158,7 @@ public GenerationTransaction(IExtraNonce extraNonce, IDaemonClient daemonClient,
                             TimeHelpers.NowInUnixTimestamp(),
                             (byte) extraNonce.ExtraNoncePlaceholder.Length,
-                            "/CoiniumServ/")
+                            "/Mined by bit2pool.com/") //Changed a mined message from CoiniumServ to mining pool name.
```
don't change this please on the main repo.

jeong760 replied:
@bonesoul, Yes, I will return it back..

jeong760 committed:
Update GenerationTransaction.cs

jeong760 committed:
Added a Stronghands icon

jeong760 committed:
Added a Stonghands-example.json

jeong760 committed:
Added a Stronghands.json

jeong760 committed:
Update HashAlgorithmStatistics.cs

Change a code for  public Int32 MinerCount { get; private set; } to get a correct hash rate from per miners

jeong760 committed:
Delete config-example.json

Delete config-example.json and will be added to a /CoiniumServ/config folder.

jeong760 committed:
Add a config-exmaple.json 

Add a config-example.json righ location.
